### ISSUE TITLE
Unexpected Warning Output / Wrong Url

### ISSUE THREAD

**Issue by claudiospizzi** [ironmansoftware/universal-automation#111]
#### Describe the Issue

When invoking the following simple script in Universal Dashboard, I'm getting two warnings in the output stream:

```powershell
param ($Name)
Write-Host "Hello, $Name!"
```

Warning messages, via API and in the dashboard.

```powershell
C:\> Get-UAJobOutput -ComputerName https://powershell-api.domain.local -JobId 1

Id        : 1
Message   : Call failed. An error occurred while sending the request. GET http://localhost:10000/api/v1/Job/1
Type      : Warning
Data      :
Job       : UniversalAutomation.Job
Timestamp : 28.02.2020 01:02:55

Id        : 2
Message   : Call failed. An error occurred while sending the request. GET http://localhost:10000/api/v1/Script/1
Type      : Warning
Data      :
Job       : UniversalAutomation.Job
Timestamp : 28.02.2020 01:02:57

Id        : 3
Message   : Hello, Claudio!
Type      : Information
Data      :
Job       : UniversalAutomation.Job
Timestamp : 28.02.2020 01:02:57
```

#### To Reproduce

Steps to reproduce the behavior: (Example)

Setup:
Hosting UniversalAutomation and UniversalAutomation.Dashboard on an IIS:
- UniversalAutomation: https://powershell-api.domain.local (Port: 443 with Host Header)
- UniversalAutomation.Dashboard: https://powershell.domain.local (Port: 443 with Host Header)

Start Skript UniversalAutomation;
```powershell
Import-Module UniversalAutomation
New-Item "C:\inetpub\wwwroot\PowerShell-API\Repo" -ItemType Directory -ErrorAction SilentlyContinue
Start-UAServer -InProcess -ConnectionString "C:\inetpub\wwwroot\PowerShell-API\database.db"  -RepositoryPath 'C:\inetpub\wwwroot\PowerShell-API\Repo'
```

Start Skript UniversalAutomation.Dashbaord:
```powershell
Import-Module UniversalAutomation.Dashboard
Connect-UAServer -ComputerName https://powershell-api.domain.local
$Dashboard = New-UADashboard -ComputerName https://powershell-api.domain.local
Start-UDDashboard -Dashboard $Dashboard -Wait
```

Create the script in the dashboard and invoke it.

The part localhost and port are somewhere hardcoded? Or not respected if we run it within IIS?

#### Expected behavior

No warnings, just one information output.

#### Screenshots

![image](https://user-images.githubusercontent.com/1934246/75498496-a338e400-59c7-11ea-9a91-b97ff7162809.png)

#### Version Information

- **Host Operating System**: Windows Server 2016
- **Host PowerShell Version**: 5.1
- **Universal Automation Version**: 1.0.1
- **Universal Dashboard Version**: 2.9.0
- **UD Hosting Method**: IIS

**Comment by claudiospizzi** [comment]
@adamdriscoll Did you have any chance to check that? I think the issue with the wrong url also leads to additional problems, like the following:

I'm trying to return some PSCustomObject's to the pipeline:
![image](https://user-images.githubusercontent.com/1934246/76202997-20e5c680-61f6-11ea-9cff-c6280ff65462.png)
![image](https://user-images.githubusercontent.com/1934246/76203026-36f38700-61f6-11ea-949d-2fd5b38a8ad9.png)

But the pipeline output is empty:
![image](https://user-images.githubusercontent.com/1934246/76202917-f3991880-61f5-11ea-98f4-cee368bd2ccc.png)

And if I try to download the objects, it throw's an error:
![image](https://user-images.githubusercontent.com/1934246/76202939-fdbb1700-61f5-11ea-9957-cda1e9a795f5.png)

But getting it from the UA itself, it works:
![image](https://user-images.githubusercontent.com/1934246/76203228-93ef3d00-61f6-11ea-9a6f-36643c3b2ccc.png)

**Comment by adamdriscoll** [comment]
@claudiospizzi - Yep. I found the source of this issue and it should be resolved in v1.1.0. The problem was the API was failing to be called correctly which resulted in a whole bunch of other issues downstream. Hopefully, we will be releasing v1.1 this week.
